test/e2e/testing-manifests/haproxyrc.yaml

```diff
@@ -0,0 +1,51 @@
+kind: ReplicationController
```

Comment:
We've conventionally stayed away from doing this. But I don't think all loadbalancers in contrib need to be duplicated in examples/, and I want to reduce the overhead of adding a new loadbalancer to just defining the yaml for it in this dir (any maybe a tiny change to the test, but nothing more).